pkg/storage/s3/s3.go

```diff
@@ -81,15 +82,27 @@ func NewImageStore(rootDir string, cacheDir string, gc bool, gcDelay time.Durati
 		dedupe:           dedupe,
 	}
 
-	cachePath := path.Join(cacheDir, CacheDBName+storage.DBExtensionName)
+	databaseType := os.Getenv("ZOT_CACHEDB_TYPE")
 
-	if dedupe {
-		imgStore.cache = storage.NewCache(cacheDir, CacheDBName, false, log)
-	} else {
-		// if dedupe was used in previous runs use it to serve blobs correctly
-		if _, err := os.Stat(cachePath); err == nil {
-			log.Info().Str("cache path", cachePath).Msg("found cache database")
-			imgStore.cache = storage.NewCache(cacheDir, CacheDBName, false, log)
+	switch databaseType {
+	case "dynamodb":
+		log.Debug().Msg("Using dynamodb")
+		imgStore.cache, _ = database.Create("dynamodb", dynamodatabase.DynamoDBDriverParameters{}, log)
```

Comment:
I don't think there should be any difference between creating a dynamodb cache and a boltdb cache.